unity-renderer/Assets/Desktop/Scenes/LoginSceneController.cs

```diff
@@ -1,16 +1,37 @@
-using System.Collections;
-using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using DCL.Providers;
 using Login;
+using System;
+using System.Collections;
+using System.Threading;
 using UnityEngine;
+using Environment = DCL.Environment;
 
 public class LoginSceneController : MonoBehaviour
 {
     private LoginHUDController _loginHUDController;
 
+    private CancellationTokenSource cancellationTokenSource;
+
     // Start is called before the first frame update
-    void Start()
+    private IEnumerator Start()
+    {
+        var assetProvider = Environment.i.serviceLocator.Get<IAddressableResourceProvider>();
+        cancellationTokenSource = new CancellationTokenSource();
+
+        yield return assetProvider.Instantiate<ILoginHUDView>("LoginHUD", cancellationToken: cancellationTokenSource.Token)
```

Comment:
Missing the retry here as well. Maybe its something that we can embedded in the AddresableProvider? Because we'll be adding retries everywhere. WDYT?